ext/date/date_core.c

```diff
@@ -2473,6 +2473,8 @@ date_s__valid_jd_p(int argc, VALUE *argv, VALUE klass)
  *
  *   Date.valid_jd?(2451944) # => true
  *
+ * See {Argument start}[rdoc-ref:Date@Argument+start].
```

Comment:
I think we should use something like

```suggestion
 * See {Argument +start+}[rdoc-ref:Date@Argument+start].
```

to match the title it links to.

Reply:
Fixed (17 places).

Reply:
I just checked this, it looks like RDoc doesn't handle formatting inside links, so it ends up being broken. Sorry about that. Maybe we can revert back to what it was like originally or something like `{Argument "start"}[rdoc-ref:Date@Argument+start]` instead?

Reply:
:+1: I like `See argument {start}[rdoc-ref:Date@Argument+start]`

Reply:
Done.